DogServiceExample/src/main/java/com/common/json/JsonHelper.java

```diff
@@ -15,4 +16,10 @@ public static String mapToJson(Map<Integer, Dog> map) {
             map = new HashMap<Integer, Dog>();
         return gson.toJson(map);
     }
+
+    public static String JsonObjectToJson(JsonObject jsonObject) {
+        if (jsonObject == null)
```

Comment:
Are this null-checks important? It seems to me that if you pass `null` here it is a logical error in your program, should handle it appropriately in some other place